Introduce EventTargetInlines.h and move inline functions there

rniwa opened the pull request:
#### 02ffb3d0d1d2c07f9640571915fad69d7a9614a7
<pre>
Introduce EventTargetInlines.h and move inline functions there
<a href="https://bugs.webkit.org/show_bug.cgi?id=292245">https://bugs.webkit.org/show_bug.cgi?id=292245</a>

Reviewed by Anne van Kesteren.

This PR extracts EventTargetInlines.h out of EventTarget.h.

* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/WebGPU/GPUDevice.h:
* Source/WebCore/Modules/applepay/ApplePaySession.cpp:
* Source/WebCore/Modules/applepay/ApplePaySession.h:
* Source/WebCore/Modules/async-clipboard/Clipboard.cpp:
* Source/WebCore/Modules/audiosession/DOMAudioSession.cpp:
* Source/WebCore/Modules/audiosession/DOMAudioSession.h:
* Source/WebCore/Modules/cookie-store/CookieStore.cpp:
* Source/WebCore/Modules/encryptedmedia/MediaKeySession.h:
* Source/WebCore/Modules/encryptedmedia/legacy/WebKitMediaKeySession.h:
* Source/WebCore/Modules/indexeddb/IDBDatabase.cpp:
* Source/WebCore/Modules/indexeddb/IDBDatabase.h:
* Source/WebCore/Modules/indexeddb/IDBRequest.cpp:
* Source/WebCore/Modules/indexeddb/IDBTransaction.h:
* Source/WebCore/Modules/mediarecorder/MediaRecorder.h:
* Source/WebCore/Modules/mediasession/MediaSessionCoordinator.cpp:
* Source/WebCore/Modules/mediasession/MediaSessionCoordinator.h:
* Source/WebCore/Modules/mediasource/MediaSource.cpp:
* Source/WebCore/Modules/mediasource/SourceBuffer.h:
* Source/WebCore/Modules/mediasource/SourceBufferList.h:
* Source/WebCore/Modules/mediastream/MediaDevices.cpp:
* Source/WebCore/Modules/mediastream/MediaDevices.h:
* Source/WebCore/Modules/mediastream/MediaStream.cpp:
* Source/WebCore/Modules/mediastream/MediaStream.h:
* Source/WebCore/Modules/mediastream/MediaStreamTrack.cpp:
* Source/WebCore/Modules/mediastream/MediaStreamTrack.h:
* Source/WebCore/Modules/mediastream/RTCDTMFSender.h:
* Source/WebCore/Modules/mediastream/RTCDataChannel.h:
* Source/WebCore/Modules/mediastream/RTCDtlsTransport.cpp:
* Source/WebCore/Modules/mediastream/RTCDtlsTransport.h:
* Source/WebCore/Modules/mediastream/RTCIceTransport.cpp:
* Source/WebCore/Modules/mediastream/RTCIceTransport.h:
* Source/WebCore/Modules/mediastream/RTCPeerConnection.h:
* Source/WebCore/Modules/mediastream/RTCRtpSFrameTransform.cpp:
* Source/WebCore/Modules/mediastream/RTCRtpSFrameTransform.h:
* Source/WebCore/Modules/mediastream/RTCSctpTransport.cpp:
* Source/WebCore/Modules/mediastream/RTCSctpTransport.h:
* Source/WebCore/Modules/notifications/Notification.cpp:
* Source/WebCore/Modules/notifications/Notification.h:
* Source/WebCore/Modules/paymentrequest/PaymentRequest.cpp:
* Source/WebCore/Modules/paymentrequest/PaymentRequest.h:
* Source/WebCore/Modules/paymentrequest/PaymentResponse.h:
* Source/WebCore/Modules/permissions/PermissionStatus.cpp:
* Source/WebCore/Modules/permissions/PermissionStatus.h:
* Source/WebCore/Modules/pictureinpicture/PictureInPictureWindow.h:
* Source/WebCore/Modules/remoteplayback/RemotePlayback.h:
* Source/WebCore/Modules/screen-wake-lock/WakeLockSentinel.cpp:
* Source/WebCore/Modules/screen-wake-lock/WakeLockSentinel.h:
* Source/WebCore/Modules/speech/SpeechRecognition.cpp:
* Source/WebCore/Modules/speech/SpeechRecognition.h:
* Source/WebCore/Modules/speech/SpeechSynthesis.cpp:
* Source/WebCore/Modules/speech/SpeechSynthesis.h:
* Source/WebCore/Modules/speech/SpeechSynthesisUtterance.cpp:
* Source/WebCore/Modules/speech/SpeechSynthesisUtterance.h:
* Source/WebCore/Modules/webaudio/AudioNode.cpp:
* Source/WebCore/Modules/webaudio/AudioScheduledSourceNode.cpp:
* Source/WebCore/Modules/webaudio/BaseAudioContext.cpp:
* Source/WebCore/Modules/webaudio/ScriptProcessorNode.cpp:
* Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.h:
* Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoDecoder.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoEncoder.h:
* Source/WebCore/Modules/websockets/WebSocket.cpp:
* Source/WebCore/Modules/webxr/WebXRLayer.h:
* Source/WebCore/Modules/webxr/WebXRSession.h:
* Source/WebCore/Modules/webxr/WebXRSpace.h:
* Source/WebCore/Modules/webxr/WebXRSystem.h:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
* Source/WebCore/accessibility/AccessibilitySVGObject.cpp:
* Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
* Source/WebCore/animation/WebAnimation.h:
* Source/WebCore/bindings/js/JSEventTargetCustom.h:
* Source/WebCore/bindings/scripts/InFilesCompiler.pm:
(generateInterfacesHeader):
* Source/WebCore/css/FontFaceSet.h:
* Source/WebCore/css/MediaQueryList.cpp:
* Source/WebCore/css/MediaQueryList.h:
* Source/WebCore/dom/AbortSignal.cpp:
* Source/WebCore/dom/AbortSignal.h:
* Source/WebCore/dom/BroadcastChannel.cpp:
* Source/WebCore/dom/BroadcastChannel.h:
* Source/WebCore/dom/EventContext.cpp:
* Source/WebCore/dom/EventTarget.cpp:
(WebCore::EventTarget::hasAnyEventListeners const):
* Source/WebCore/dom/EventTarget.h:
(WebCore::EventTarget::eventTargetData const): Deleted.
(WebCore::EventTarget::eventTargetData): Deleted.
(WebCore::EventTarget::eventTargetDataConcurrently): Deleted.
(WebCore::EventTarget::enumerateEventListenerTypes const): Deleted.
(WebCore::EventTarget::containsMatchingEventListener const): Deleted.
(WebCore::EventTarget::ensureEventTargetData): Deleted.
(WebCore::EventTarget::hasEventListeners const): Deleted.
(WebCore::EventTarget::hasAnyEventListeners const): Deleted.
(WebCore::EventTarget::hasCapturingEventListeners): Deleted.
(WebCore::EventTarget::visitJSEventListeners): Deleted.
* Source/WebCore/dom/EventTargetFactory.in:
* Source/WebCore/dom/EventTargetInlines.h: Added.
(WebCore::EventTarget::hasEventListeners const):
(WebCore::EventTarget::hasCapturingEventListeners):
(WebCore::EventTarget::eventTargetData const):
(WebCore::EventTarget::eventTargetData):
(WebCore::EventTarget::eventTargetDataConcurrently):
(WebCore::EventTarget::ensureEventTargetData):
(WebCore::EventTarget::enumerateEventListenerTypes const):
(WebCore::EventTarget::containsMatchingEventListener const):
(WebCore::EventTarget::visitJSEventListeners):
* Source/WebCore/dom/MessagePort.cpp:
* Source/WebCore/dom/MessagePort.h:
* Source/WebCore/dom/Node.cpp:
* Source/WebCore/editing/TextManipulationController.cpp:
* Source/WebCore/fileapi/FileReader.h:
* Source/WebCore/html/HTMLCanvasElement.cpp:
* Source/WebCore/html/HTMLMediaElement.cpp:
* Source/WebCore/html/HTMLTrackElement.cpp:
* Source/WebCore/html/MediaController.h:
* Source/WebCore/html/OffscreenCanvas.h:
* Source/WebCore/html/closewatcher/CloseWatcher.cpp:
* Source/WebCore/html/closewatcher/CloseWatcher.h:
* Source/WebCore/html/track/AudioTrackList.cpp:
* Source/WebCore/html/track/TextTrack.h:
* Source/WebCore/html/track/TextTrackCue.h:
* Source/WebCore/html/track/TextTrackList.cpp:
* Source/WebCore/html/track/VideoTrackList.cpp:
* Source/WebCore/inspector/InspectorAuditDOMObject.cpp:
* Source/WebCore/inspector/agents/InspectorCSSAgent.cpp:
* Source/WebCore/loader/appcache/DOMApplicationCache.h:
* Source/WebCore/page/DOMWindow.h:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/EventSource.h:
* Source/WebCore/page/LocalDOMWindow.cpp:
* Source/WebCore/page/LocalDOMWindow.h:
* Source/WebCore/page/Navigation.cpp:
* Source/WebCore/page/NavigationHistoryEntry.cpp:
* Source/WebCore/page/Performance.h:
* Source/WebCore/page/PointerCaptureController.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/ScreenOrientation.cpp:
* Source/WebCore/page/ScreenOrientation.h:
* Source/WebCore/page/VisualViewport.cpp:
* Source/WebCore/page/ios/FrameIOS.mm:
* Source/WebCore/style/StyleAdjuster.cpp:
* Source/WebCore/svg/SVGElement.cpp:
* Source/WebCore/svg/SVGUseElement.cpp:
* Source/WebCore/workers/DedicatedWorkerGlobalScope.cpp:
* Source/WebCore/workers/Worker.h:
* Source/WebCore/workers/service/ServiceWorker.cpp:
* Source/WebCore/workers/service/ServiceWorkerContainer.h:
* Source/WebCore/workers/service/ServiceWorkerGlobalScope.cpp:
* Source/WebCore/workers/service/ServiceWorkerRegistration.cpp:
* Source/WebCore/workers/service/background-fetch/BackgroundFetchRegistration.h:
* Source/WebCore/workers/shared/SharedWorker.cpp:
* Source/WebCore/workers/shared/SharedWorkerGlobalScope.h:
* Source/WebCore/worklets/WorkletGlobalScope.h:
* Source/WebCore/xml/XMLHttpRequest.cpp:
* Source/WebCore/xml/XMLHttpRequest.h:
* Source/WebCore/xml/XMLHttpRequestProgressEventThrottle.cpp:
* Source/WebCore/xml/XMLHttpRequestUpload.cpp:
* Source/WebCore/xml/XMLHttpRequestUpload.h:

Canonical link: <a href="https://commits.webkit.org/294283@main">https://commits.webkit.org/294283@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/83bee42005830d09fde0e8dc250bc419472d946d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/101282 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/20944 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/11247 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/106430 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/51909 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/103323 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/21253 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/29439 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/77139 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/34170 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/104289 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/16389 "Found 1 new test failure: fast/forms/ios/remove-view-after-focus.html (failure)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/91479 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/57486 "Found 1 new API test failure: /WPE/TestCookieManager:/webkit/WebKitCookieManager/replace-get-all-cookies (failure)") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/16205 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/9490 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/51257 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/86135 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/9565 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/108786 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/28410 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/20901 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/86119 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/28772 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/87687 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/85675 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/30393 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/8101 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/22509 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/16491 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/28340 "Built successfully") | [❌ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/33615 "Failed to build and analyze WebKit") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/28152 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/31472 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/29710 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rniwa commented on Source/WebCore/dom/EventTargetFactory.in:
```diff
@@ -2,6 +2,7 @@ namespace="EventTarget"
 factoryFunction=toJS
 useNamespaceAsSuffix=true
 
+
```
This is necessary to force a rebuild of `EventTargetInterfaces.h`.

annevk replied:
Hmm, it seems like dependencies are not setup correctly then in the build system somewhere? Might be good to file a follow-up for this.

rniwa replied:
Tracking this as https://bugs.webkit.org/show_bug.cgi?id=292282.

rniwa committed:
Introduce EventTargetInlines.h and move inline functions there
https://bugs.webkit.org/show_bug.cgi?id=292245

Reviewed by Anne van Kesteren.

This PR extracts EventTargetInlines.h out of EventTarget.h.

* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/WebGPU/GPUDevice.h:
* Source/WebCore/Modules/applepay/ApplePaySession.cpp:
* Source/WebCore/Modules/applepay/ApplePaySession.h:
* Source/WebCore/Modules/async-clipboard/Clipboard.cpp:
* Source/WebCore/Modules/audiosession/DOMAudioSession.cpp:
* Source/WebCore/Modules/audiosession/DOMAudioSession.h:
* Source/WebCore/Modules/cookie-store/CookieStore.cpp:
* Source/WebCore/Modules/encryptedmedia/MediaKeySession.h:
* Source/WebCore/Modules/encryptedmedia/legacy/WebKitMediaKeySession.h:
* Source/WebCore/Modules/indexeddb/IDBDatabase.cpp:
* Source/WebCore/Modules/indexeddb/IDBDatabase.h:
* Source/WebCore/Modules/indexeddb/IDBRequest.cpp:
* Source/WebCore/Modules/indexeddb/IDBTransaction.h:
* Source/WebCore/Modules/mediarecorder/MediaRecorder.h:
* Source/WebCore/Modules/mediasession/MediaSessionCoordinator.cpp:
* Source/WebCore/Modules/mediasession/MediaSessionCoordinator.h:
* Source/WebCore/Modules/mediasource/MediaSource.cpp:
* Source/WebCore/Modules/mediasource/SourceBuffer.h:
* Source/WebCore/Modules/mediasource/SourceBufferList.h:
* Source/WebCore/Modules/mediastream/MediaDevices.cpp:
* Source/WebCore/Modules/mediastream/MediaDevices.h:
* Source/WebCore/Modules/mediastream/MediaStream.cpp:
* Source/WebCore/Modules/mediastream/MediaStream.h:
* Source/WebCore/Modules/mediastream/MediaStreamTrack.cpp:
* Source/WebCore/Modules/mediastream/MediaStreamTrack.h:
* Source/WebCore/Modules/mediastream/RTCDTMFSender.h:
* Source/WebCore/Modules/mediastream/RTCDataChannel.h:
* Source/WebCore/Modules/mediastream/RTCDtlsTransport.cpp:
* Source/WebCore/Modules/mediastream/RTCDtlsTransport.h:
* Source/WebCore/Modules/mediastream/RTCIceTransport.cpp:
* Source/WebCore/Modules/mediastream/RTCIceTransport.h:
* Source/WebCore/Modules/mediastream/RTCPeerConnection.h:
* Source/WebCore/Modules/mediastream/RTCRtpSFrameTransform.cpp:
* Source/WebCore/Modules/mediastream/RTCRtpSFrameTransform.h:
* Source/WebCore/Modules/mediastream/RTCSctpTransport.cpp:
* Source/WebCore/Modules/mediastream/RTCSctpTransport.h:
* Source/WebCore/Modules/notifications/Notification.cpp:
* Source/WebCore/Modules/notifications/Notification.h:
* Source/WebCore/Modules/paymentrequest/PaymentRequest.cpp:
* Source/WebCore/Modules/paymentrequest/PaymentRequest.h:
* Source/WebCore/Modules/paymentrequest/PaymentResponse.h:
* Source/WebCore/Modules/permissions/PermissionStatus.cpp:
* Source/WebCore/Modules/permissions/PermissionStatus.h:
* Source/WebCore/Modules/pictureinpicture/PictureInPictureWindow.h:
* Source/WebCore/Modules/remoteplayback/RemotePlayback.h:
* Source/WebCore/Modules/screen-wake-lock/WakeLockSentinel.cpp:
* Source/WebCore/Modules/screen-wake-lock/WakeLockSentinel.h:
* Source/WebCore/Modules/speech/SpeechRecognition.cpp:
* Source/WebCore/Modules/speech/SpeechRecognition.h:
* Source/WebCore/Modules/speech/SpeechSynthesis.cpp:
* Source/WebCore/Modules/speech/SpeechSynthesis.h:
* Source/WebCore/Modules/speech/SpeechSynthesisUtterance.cpp:
* Source/WebCore/Modules/speech/SpeechSynthesisUtterance.h:
* Source/WebCore/Modules/webaudio/AudioNode.cpp:
* Source/WebCore/Modules/webaudio/AudioScheduledSourceNode.cpp:
* Source/WebCore/Modules/webaudio/BaseAudioContext.cpp:
* Source/WebCore/Modules/webaudio/ScriptProcessorNode.cpp:
* Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.h:
* Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoDecoder.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoEncoder.h:
* Source/WebCore/Modules/websockets/WebSocket.cpp:
* Source/WebCore/Modules/webxr/WebXRLayer.h:
* Source/WebCore/Modules/webxr/WebXRSession.h:
* Source/WebCore/Modules/webxr/WebXRSpace.h:
* Source/WebCore/Modules/webxr/WebXRSystem.h:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
* Source/WebCore/accessibility/AccessibilitySVGObject.cpp:
* Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
* Source/WebCore/animation/WebAnimation.h:
* Source/WebCore/bindings/js/JSEventTargetCustom.h:
* Source/WebCore/bindings/scripts/InFilesCompiler.pm:
(generateInterfacesHeader):
* Source/WebCore/css/FontFaceSet.h:
* Source/WebCore/css/MediaQueryList.cpp:
* Source/WebCore/css/MediaQueryList.h:
* Source/WebCore/dom/AbortSignal.cpp:
* Source/WebCore/dom/AbortSignal.h:
* Source/WebCore/dom/BroadcastChannel.cpp:
* Source/WebCore/dom/BroadcastChannel.h:
* Source/WebCore/dom/EventContext.cpp:
* Source/WebCore/dom/EventTarget.cpp:
(WebCore::EventTarget::hasAnyEventListeners const):
* Source/WebCore/dom/EventTarget.h:
(WebCore::EventTarget::eventTargetData const): Deleted.
(WebCore::EventTarget::eventTargetData): Deleted.
(WebCore::EventTarget::eventTargetDataConcurrently): Deleted.
(WebCore::EventTarget::enumerateEventListenerTypes const): Deleted.
(WebCore::EventTarget::containsMatchingEventListener const): Deleted.
(WebCore::EventTarget::ensureEventTargetData): Deleted.
(WebCore::EventTarget::hasEventListeners const): Deleted.
(WebCore::EventTarget::hasAnyEventListeners const): Deleted.
(WebCore::EventTarget::hasCapturingEventListeners): Deleted.
(WebCore::EventTarget::visitJSEventListeners): Deleted.
* Source/WebCore/dom/EventTargetFactory.in:
* Source/WebCore/dom/EventTargetInlines.h: Added.
(WebCore::EventTarget::hasEventListeners const):
(WebCore::EventTarget::hasCapturingEventListeners):
(WebCore::EventTarget::eventTargetData const):
(WebCore::EventTarget::eventTargetData):
(WebCore::EventTarget::eventTargetDataConcurrently):
(WebCore::EventTarget::ensureEventTargetData):
(WebCore::EventTarget::enumerateEventListenerTypes const):
(WebCore::EventTarget::containsMatchingEventListener const):
(WebCore::EventTarget::visitJSEventListeners):
* Source/WebCore/dom/MessagePort.cpp:
* Source/WebCore/dom/MessagePort.h:
* Source/WebCore/dom/Node.cpp:
* Source/WebCore/editing/TextManipulationController.cpp:
* Source/WebCore/fileapi/FileReader.h:
* Source/WebCore/html/HTMLCanvasElement.cpp:
* Source/WebCore/html/HTMLMediaElement.cpp:
* Source/WebCore/html/HTMLTrackElement.cpp:
* Source/WebCore/html/MediaController.h:
* Source/WebCore/html/OffscreenCanvas.h:
* Source/WebCore/html/closewatcher/CloseWatcher.cpp:
* Source/WebCore/html/closewatcher/CloseWatcher.h:
* Source/WebCore/html/track/AudioTrackList.cpp:
* Source/WebCore/html/track/TextTrack.h:
* Source/WebCore/html/track/TextTrackCue.h:
* Source/WebCore/html/track/TextTrackList.cpp:
* Source/WebCore/html/track/VideoTrackList.cpp:
* Source/WebCore/inspector/InspectorAuditDOMObject.cpp:
* Source/WebCore/inspector/agents/InspectorCSSAgent.cpp:
* Source/WebCore/loader/appcache/DOMApplicationCache.h:
* Source/WebCore/page/DOMWindow.h:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/EventSource.h:
* Source/WebCore/page/LocalDOMWindow.cpp:
* Source/WebCore/page/LocalDOMWindow.h:
* Source/WebCore/page/Navigation.cpp:
* Source/WebCore/page/NavigationHistoryEntry.cpp:
* Source/WebCore/page/Performance.h:
* Source/WebCore/page/PointerCaptureController.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/ScreenOrientation.cpp:
* Source/WebCore/page/ScreenOrientation.h:
* Source/WebCore/page/VisualViewport.cpp:
* Source/WebCore/page/ios/FrameIOS.mm:
* Source/WebCore/style/StyleAdjuster.cpp:
* Source/WebCore/svg/SVGElement.cpp:
* Source/WebCore/svg/SVGUseElement.cpp:
* Source/WebCore/workers/DedicatedWorkerGlobalScope.cpp:
* Source/WebCore/workers/Worker.h:
* Source/WebCore/workers/service/ServiceWorker.cpp:
* Source/WebCore/workers/service/ServiceWorkerContainer.h:
* Source/WebCore/workers/service/ServiceWorkerGlobalScope.cpp:
* Source/WebCore/workers/service/ServiceWorkerRegistration.cpp:
* Source/WebCore/workers/service/background-fetch/BackgroundFetchRegistration.h:
* Source/WebCore/workers/shared/SharedWorker.cpp:
* Source/WebCore/workers/shared/SharedWorkerGlobalScope.h:
* Source/WebCore/worklets/WorkletGlobalScope.h:
* Source/WebCore/xml/XMLHttpRequest.cpp:
* Source/WebCore/xml/XMLHttpRequest.h:
* Source/WebCore/xml/XMLHttpRequestProgressEventThrottle.cpp:
* Source/WebCore/xml/XMLHttpRequestUpload.cpp:
* Source/WebCore/xml/XMLHttpRequestUpload.h:

Canonical link: https://commits.webkit.org/294283@main
